data/json/items/gunmod/underbarrel.json

```diff
@@ -167,6 +167,25 @@
     "description": "A grip placed forward on the barrel allows for greater control.  Not as effective as a bipod but usable under all conditions.  This one was modified and customized to mount on pretty much any weapon, if you so want.",
     "mod_targets": [ "pistol", "smg", "rifle", "shotgun", "launcher", "crossbow" ]
   },
+  {
+    "id": "wooden_grip",
+    "type": "GUNMOD",
+    "name": { "str": "wood forward grip" },
+    "description": "A wooden grip placed forward on the barrel allows for greater control.  Not as effective as a bipod but usable under all conditions.  This one was cleverly handmade by a desperate survivor or miserly enthusiast, it cannot be removed after installation.",
+    "weight": "94 g",
+    "volume": "119 ml",
+    "price": 200,
+    "price_postapoc": 250,
+    "install_time": "15 m",
+    "material": [ "wood" ],
+    "symbol": ":",
+    "color": "brown",
+    "location": "underbarrel",
+    "mod_targets": [ "pistol", "smg", "rifle", "shotgun", "launcher", "crossbow" ],
+    "handling_modifier": 6,
+    "min_skills": [ [ "weapon", 3 ] ],
```

Comment:
Can the install requirements on this be lowered too 👉👈

Reply:
As far as I recall, you should be able to install it with lower skill, just with a low chance of damaging the weapon. Nailing an attachment onto your weapon without a chance of damage seems like it would be in the realm of skill 3.